src/terra-dev-site/guide/Internationalization/InternationalizationIntro.a.guide.md

```diff
@@ -0,0 +1,85 @@
+
+# Getting Started
+Install [terra-base](https://www.npmjs.com/package/terra-base) and [terra-toolkit](https://www.npmjs.com/package/terra-toolkit).
+
+If you're using npm, run the following command:
+
+```
+npm install terra-base terra-toolkit
+```
+
+If you're using yarn, run the following command:
+
+```
+yarn add terra-base terra-toolkit
+```
+
+# Terra Base
+The first part of setting up internalization and translations in your app requires installing and configuring terra-base. This component is designed to manage loading locale information such as translations and locale data, handling locale switching and loading sate, and combining customized translation messages from an application with Terra UI component translations.
+
+**Note:** *This component should be your root component and wrap all components in your app. This is to help ensure that the entire application is within the same configured i18n context provided by [react-intl's `<IntlProvider />`](https://github.com/yahoo/react-intl/wiki/Components#intlprovider). The terra-base component provides an abstraction around react-intl's `<IntlProvider />` component which allows code splitting of translations per locale.*
+
+You can [read more about react-intl's API here](https://github.com/yahoo/react-intl/wiki/API).
+
+If your app doesn't have any custom translations, you can set up and use terra-base as follows:
+
+```jsx
+import Base from 'terra-base';
+import App from './App'; // Your custom app component
+
+// This value could be sent from the server as well
+const locale = (navigator.languages && navigator.languages[0])
+               || navigator.language
+               || navigator.userLanguage
+               || 'en';
+
+<Base locale={locale}>
+  <App />
+</Base>
+```
+
+This will load the `en` translations for any Terra UI components you use in your app.
+
+**Note:** *This requires running terra-toolkit's aggregate translation tool to fully work. See the terra-toolkit's aggregate translation section below for more info.*
+
+If your app has custom translations outside of the ones provided by Terra UI, you can set up and use terra-base as follows:
+
+```jsx
+import Base from 'terra-base';
+import App from './App'; // Your custom app component
+import appTranslations from './aggregated-translations/en.js'; // Your aggregated app translations
+
+// This value could be sent from the server as well
+const locale = (navigator.languages && navigator.languages[0])
+               || navigator.language
+               || navigator.userLanguage
+               || 'en';
+
+<Base locale={locale} customMessages={appTranslations}>
```

Comment:
Even though custom translations can be provided via the prop, do we want to recommend this approach or should we recommend setting up a local translations directory with a locale.json file for each locale in the project so translations are aggregated during tt-aggregate-translations?

I guess, did we intend the customMessages prop to be used for adding translations specific to an app or for overriding our defaults. For example, instead of Terra's loading message, they overriding it with their own ?

Reply:
My thought is customMessages prop is used for adding app translations in addition to the ones we provide for terra components out of the box.